docs/api/portable-stories.md

```diff
@@ -0,0 +1,376 @@
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/portable-stories-with-loaders.ts.mdx',
+    'vue/portable-stories-with-loaders.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+### 3. Render
+
+The story has been prepared and can be rendered. You pass it into the rendering mechanism of your choice (e.g. Testing Library render function, Vue test utils mount function, etc.)
+
+👉 For this, you use the [`composeStories`](#composestories) or [`composeStory`](#composestory) API. The composed Story is a renderable component that can be passed to your rendering mechanism.
+
+### 4. Play (optional)
+
+Once the component is rendered, you execute the play function which will contain interactions and assertions. This only applies for stories that use the [play function](../essentials/interactions.md#play-function-for-interactions).
+
+👉 For this, you use the [`composeStories`](#composestories) or [`composeStory`](#composestory) API. The composed Story will return a `play` method to be called **after** it has rendered.
+
+For example, your story might have a play function that prepares your component's desired state: in this case, clicking on a button to display a modal. In portable stories, the play function does not run automatically – you have to call it yourself. The play function needs a `canvasElement`, which should be passed by you. A `canvasElement` is the HTML element which wraps your component. Each testing utility provides different ways to retrieve such element, but here's how to do it with Testing Library:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/portable-stories-with-play-function.ts.mdx',
+    'vue/portable-stories-with-play-function.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<Callout variant="info">
+
+If your play function contains assertions (e.g. `expect` calls), your test will fail when those assertions fail.
+
+</Callout>
+
+## Overriding globals
+
+If your stories behave differently based on [globals](../essentials/toolbars-and-globals.md#globals), such as they render text in English or Spanish, and such behavior can be changed via the toolbar items, you can achieve similar functionality in portable stories by overriding project annotations when composing a story:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/portable-stories-override-globals.ts.mdx',
+    'vue/portable-stories-override-globals.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+## Component testing integrations
+
+### Playwright CT
+
+Portable stories work in [Playwright CT](https://playwright.dev/docs/test-components), although with a few modifications, as well as different mounting mechanism.
```

Comment:
Maybe we use an image like so
<img width="1251" alt="image" src="https://github.com/storybookjs/storybook/assets/1671563/ca7bd677-0816-46c1-9cac-70bcbfeeb99d">

Reply:
@cdedreuille — Here's another one we'll need. We accidentally flipped the order the first two boxes contained within `playwright test`. It should match the order of the other image.